source/dub/dub.d

```diff
@@ -190,9 +224,12 @@ class Dub {
 	deprecated void shutdown() {}
 	deprecated void cleanCaches() {}
 
-	/// Loads the package from the current working directory as the main
-	/// project package.
-	void loadPackageFromCwd()
+	deprecated("Use loadPackage instead.")
+	alias loadPackageFromCwd = loadPackage;
```

Comment:
My plan is to remove all deprecated symbols for the 1.0.0 release, so that we can start with a clean slate. So instead of a date, I'd mention the version, but I'll add that for all deprecations throughout the API.